draft-kuepau-iab-privacy-partitioning.md

```diff
@@ -454,7 +436,42 @@ about individual client data.
 
 # Applying Privacy Partioning
 
-TODO: talk about how to think about new protocols that are using partitioning, the models that we derive from looking at the existing ones, how you recognize a protocol that's using this, and what tools do we have to evaluate it
+Applying privacy partitioning to an existing or new system or protocol is a straightforward process.
+
+1. Identify all types of information used or exposed in a system or protocol, some of which may be user-identifying.
+1. Partition different types of user-identifying information across contexts to minimize the amount of
```

Comment:
I'm concerned that just partitioning out the data that is obviously user-identifying isn't enough; how can we capture the correlation angle too?

Reply:
I think your suggested text probably covers it, which states that two contexts must not be trivially correlated through any shared or common info.